internal/tss/store/store.go

```diff
@@ -0,0 +1,66 @@
+package store
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stellar/wallet-backend/internal/db"
+	"github.com/stellar/wallet-backend/internal/tss"
+)
+
+type Store interface {
+	UpsertTransaction(ctx context.Context, WebhookURL string, txHash string, txXDR string, status tss.RPCTXStatus) error
+	UpsertTry(ctx context.Context, transactionHash string, feeBumpTxHash string, feeBumpTxXDR string, status tss.RPCTXCode) error
+}
+
+var _ Store = (*store)(nil)
+
+type store struct {
+	DB db.ConnectionPool
+}
+
+func NewStore(db db.ConnectionPool) Store {
+	return &store{
+		DB: db,
+	}
+}
+
+func (s *store) UpsertTransaction(ctx context.Context, webhookURL string, txHash string, txXDR string, status tss.RPCTXStatus) error {
+	const q = `
+	INSERT INTO 
+		tss_transactions (transaction_hash, transaction_xdr, webhook_url, current_status)
+	VALUES
+		($1, $2, $3, $4)
+	ON CONFLICT (transaction_hash) 
+	DO UPDATE SET 
+		transaction_xdr = $2,
+		webhook_url = $3,
+    	current_status = $4,
```

Comment:
💡 When handling ON CONFLICT's you can use Postgre's EXCLUDED keywork.
Btw, seems there's something up with the indentation here.
```suggestion
		transaction_xdr = EXCLUDED.transaction_xdr,
		webhook_url = EXCLUDED.webhook_url,
    	        current_status = EXCLUDED.current_status,
```

Reply:
Nice! made this change